processor/configuration/src/test/java/com/redhat/service/smartevents/processor/sources/aws/AwsS3SourceConnectorTest.java

```diff
@@ -0,0 +1,103 @@
+package com.redhat.service.smartevents.processor.sources.aws;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.service.smartevents.infra.models.gateways.Source;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+class AwsS3SourceConnectorTest {
+
+    private static final String TEST_BUCKET = "test-bucket";
+    private static final String TEST_REGION = "test-region";
+    private static final String TEST_ACCESS_KEY = "test-access-key";
+    private static final String TEST_SECRET_KEY = "test-secret-key";
+    private static final Boolean TEST_IGNORE_BODY = true;
+    private static final Boolean TEST_DELETE_AFTER_READ = false;
+    private static final String TEST_TOPIC_NAME = "test-topic-name";
+    private static final String TEST_PREFIX = "test-prefix";
+
+    private static final String EXPECTED_PAYLOAD_JSON = "{" +
+            "   \"aws_bucket_name_or_arn\":\"" + TEST_BUCKET + "\"," +
+            "   \"aws_region\":\"" + TEST_REGION + "\"," +
+            "   \"aws_access_key\":\"" + TEST_ACCESS_KEY + "\"," +
+            "   \"aws_secret_key\":\"" + TEST_SECRET_KEY + "\"," +
+            "   \"aws_prefix\":\"" + TEST_PREFIX + "\"," +
```

Comment:
here we have "aws_prefix", while we have simply prefix in `AwsS3Source.PREFIX`, is it intentional?

Reply:
@jomarko yes these are the parameter needed by MC while in `AwsS3Source` they're our API. it's probably better to use `aws_` anyway